pkgs/os-specific/linux/vmware/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, gnugrep }:
+
+stdenv.mkDerivation rec {
+  pname = "vmware-modules";
+  version = "16.2.3-${kernel.version}";
```

```suggestion
{ lib, stdenv, fetchFromGitHub, kernel, kmod, gnugrep, vmware-workstation }:

stdenv.mkDerivation rec {
  pname = "vmware-modules";
  version = "${vmware-workstation.version}-${kernel.version}";
```

Just a small recommendation. This can be helpful when there is an update on vmware-workstation, so the kernel would already reflect the new version.

---

Done, `vmware-modules` now try to fetch workstation's version tag of `vmware-host-modules`